pam/internal/adapter/authentication.go

```diff
@@ -204,6 +205,7 @@ func (m *authenticationModel) Update(msg tea.Msg) (authModel authenticationModel
 		}
 
 	case newPasswordCheckResult:
+		log.Debugf(msg.ctx, "%T: %s", msg, msg.msg)
```

Comment:
does this (or the `log.Debugf(context.TODO(), "%T", msg)` in line 197) log the password?

Reply:
No, no... Clearly no.. Only the message (structure) type name.